actions/youtube_search_MOD.js

```diff
@@ -324,16 +324,14 @@ module.exports = {
         break
       }
       case 1: { // Playlist
-        if (apikey === undefined || apikey === '') {
-          return console.log('Please provide a valid api key.')
-        }
+        if (apikey === undefined || apikey === '') return console.error('Please provide a valid api key for YouTube Search Playlist.')
```

Comment:
Wasn't your intention to continue the action sequence instead of returning?

Reply:
Except api key is more mandatory than results being broke, but yes other than that